web_favicon/models/res_company.py

```diff
@@ -68,20 +68,22 @@ def create(self, vals_list):
     @api.model
     def _get_favicon(self):
         """Returns a local url that points to the image field of a given record."""
-        company_id = (
-            request.httprequest.cookies.get("cids")
-            if request.httprequest.cookies.get("cids")
-            else False
-        )
-        company = (
-            self.browse(int(company_id.split(",")[0])).sudo()
-            if company_id and self.browse(int(company_id.split(",")[0])).sudo().favicon
-            else False
-        )
-        if company:
-            sha = hashlib.sha512(str(company.write_date).encode("utf-8")).hexdigest()[
-                :7
-            ]
-            return f"/web/image/{self._name}/{company.id}/favicon?unique={sha}"
-        else:
-            return False
+        if not self.env.context.get("website_id"):
```

Comment:
For minimizing the diff, please add the contrary if and return early in that case, leaving the rest of the code the same.

Reply:
@pedrobaeza done!